course/frontend/has-symbol/SpecRunner.html

```diff
@@ -10,7 +10,7 @@
     <script src="vendor/jasmine/2.9.1/jasmine-html.js"></script>
     <script src="vendor/jasmine/2.9.1/boot.js"></script>
     <!-- script and specs -->
-    <script src="scripts/hasSymbol.js"></script>
+    <script src="scripts/hassymbol.js"></script>
```

Comment:
KEEP "has-symbol.js" name. "chain-case" is the right way to name a script file in js ,)